src/framework/MOM_domain_infra.F90

```diff
@@ -7,22 +7,17 @@ module MOM_domain_infra
 use MOM_cpu_clock,   only : cpu_clock_begin, cpu_clock_end
 use MOM_error_infra, only : MOM_error=>MOM_err, NOTE, WARNING, FATAL
 
-use mpp_domains_mod, only : MOM_define_layout => mpp_define_layout, mpp_get_boundary
-use mpp_domains_mod, only : MOM_define_io_domain => mpp_define_io_domain
-use mpp_domains_mod, only : MOM_define_domain => mpp_define_domains
-use mpp_domains_mod, only : domain2D, domain1D, mpp_get_data_domain, mpp_get_domain_components
-use mpp_domains_mod, only : mpp_get_compute_domain, mpp_get_global_domain
-use mpp_domains_mod, only : mpp_get_domain_extents, mpp_deallocate_domain
-use mpp_domains_mod, only : mpp_update_domains, global_field_sum => mpp_global_sum
+use mpp_domains_mod, only : domain2D, domain1D, group_pass_type => mpp_group_update_type
```

Comment:
Since `group_pass_type` is used in the code, does this need to be documented somehow?

Reply:
Because this is an opaque type whose contents are never referenced anywhere that is visible in the MOM6 code, I don't think that we need to document its contents (even if we could figure out how to do this).

Reply:
See response above, PR addresses this change.